docs/spec.md

```diff
@@ -627,20 +627,31 @@ Performs element-wise addition of two tensors `lhs` and `rhs` and produces a
 
 #### Inputs
 
-| Label | Name  | Type                                  | Constraints |
-|-------|-------|---------------------------------------|-------------|
-| (I1)  | `lhs` | tensor or per-tensor quantized tensor | (C1)        |
-| (I2)  | `rhs` | tensor or per-tensor quantized tensor | (C1)        |
+| Label | Name  | Type                       | Constraints   |
+|-------|-------|----------------------------|---------------|
+| (I1)  | `lhs` | tensor or quantized tensor | (C1-C6)       |
```

Comment:
I think this (along with @abhigunj's work on verifying quantized types) are good motivation for reworking how we spec quantized types. Namely let's make a very clear non-terminal split for QuantizedTensor, PerTensorQuantizedTensor, and PerAxisQuantizedTensor.

This can be done in a followup, but now that "per-tensor" and "quantized tensor" have different meanings, let's match it in the spec. This also will make the spec and VHLO align more closely.